emails/views.py

```diff
@@ -427,6 +434,26 @@ def _get_relay_recipient_from_message_json(message_json):
 
 def _sns_message(message_json):
     incr_if_enabled("sns_inbound_Notification_Received", 1)
+    message = None
+    try:
+        message = get_ses_message(message_json)
+    except NotImplementedError:
+        pass  # Bounce, Received, etc.
+    except Exception:
+        info_logger.exception(
+            "SES message processing error", extra={"payload": message_json}
+        )
+        return HttpResponse("OK", status=200)
```

Comment:
Should we handle for `ValueError` response from [get_ses_message_type](https://github.com/mozilla/fx-private-relay/blob/7b3eda6ec26ba144c7975e6d4709e604f2d3b17f/emails/ses.py#L248) that is called by `get_ses_message` so we can log cases when neither or both are set?

Reply:
I guess the better question to ask is what does it mean when both or neither event or notification type is set and see if logging is even necessary.